packages/checkout-storefront/src/views/OrderConfirmation/OrderConfirmation.tsx

```diff
@@ -26,7 +26,7 @@ export const OrderConfirmation = ({ orderId }: { orderId: string }) => {
         </Text>
       </header>
       <main className="order-content overflow-hidden">
-        <OrderInfo order={order} />
+        <OrderInfo />
```

Comment:
No reason we **need** to, but it IMO it's a bit better to do so. Checkout form has the `useCheckout` hook, so for once order page having `useOrder` hook makes it consistent. It also causes less prop drilling (notice that the order was further passed to `PaymentSection` from OrderInfo) and doesn't cost us any extra requests since it works based on urql cache. So I'd say "why not?" 😅

Reply:
oh ok, if that's cached then it's totally fine